hack/make-bundle-vars.mk

```diff
@@ -102,6 +102,13 @@ ROOK_BUNDLE_IMG_TAG ?= v4.15.0
 ROOK_BUNDLE_IMG_LOCATION ?= quay.io/ocs-dev
 ROOK_BUNDLE_IMG ?= $(ROOK_BUNDLE_IMG_LOCATION)/$(ROOK_BUNDLE_IMG_NAME):$(ROOK_BUNDLE_IMG_TAG)
 
+# To be changed when odf-prometheus-operator bundle exists
+PROMETHEUS_BUNDLE_NAME ?= odf-prometheus-operator
```

Comment:
The name should be same at all the places. at some places you are using `ose-prometheus-operator` and at some places `odf-prometheus-operator`

Reply:
Updated to odf-prometheus-operator